manifests/i/iQIYI/iQIYI/9.9.165.6300/iQIYI.iQIYI.locale.en-US.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: iQIYI.iQIYI
+PackageVersion: 9.9.165.6300
+PackageLocale: en-US
+Publisher: 爱奇艺
+PublisherUrl: https://www.iqiyi.com
+PrivacyUrl: https://privacy.iqiyi.com/policies
+PackageName: 爱奇艺
+License: Copyright (C) 2022 爱奇艺 All Rights Reserved
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate